apps/dashboard/src/app/(dashboard)/dashboard/connect/pay/[id]/layout.tsx

```diff
@@ -0,0 +1,61 @@
+import { TabPathLinks } from "@/components/ui/tabs";
```

Comment:
Before merging - verify that "Recipient address" input in settings tab is updating the given input properly ( do page reload and check ) 

For some reason, it's not working on locahost

Reply:
It's a bug in api server 

If I fetch a single api key using /v1/keys/<id> - it doesn't include the payoutAddress in the pay service object
but If I get all the api keys using /v1/keys/ its included

Reply:
added a workaround to always use  /v1/keys/ endpoint for now